jwst/extract_2d/tests/test_grisms.py

```diff
@@ -349,5 +349,8 @@ def test_extract_wfss_object():
     ids = [slit.source_id for slit in outmodel.slits]
     assert ids == [9, 19, 19]
```

Comment:
Is this just bogus data or can there be more than one source with the same id?

Reply:
I don't know - these are from @sosey .

Reply:
Could this be a case where more than one source is in a slitlet? E.g. in this case source 9 is in one of the 3 shutters and source 19 is in the other two? Or is that not what I'm seeing here?

Reply:
Oh, never mind. I've got NRS MSA on my mind. We're talking about WFSS data here. I suppose it's possible to have 2 sources defined in the source catalog that are close enough to one another on the sky such that when a slit is defined for one of them, it happens to also include (be contaminated by) the spectrum of the nearby source.

Reply:
There are two cases:

- two sources in one slit - we should still get two different source_id
- one source in two slits (seemingly the above case) - Shouldn't we still see two different source_ids since these are different locations on the sky (even if it's an extended source)?

Reply:
All depends on how the source detection routine works in the `source_catalog` step that runs on the direct image in a WFSS observation. I guess it's possible that with complicated extended sources it could catalog individual nearby pieces of the source with unique ID's.

Reply:
The comments in the test say it's a test of extracting 2 sources, yet it seems to be expecting a list of 3 to be returned, so that seems inconsistent. It doesn't really have anything to do with this particular update to the code, so I vote to ignore it for now and move along with getting this change merged.

Reply:
Yah, I think it's a mistake by me, I'll take a closer look and remind myself what was going on and submit a separate PR to fix?